src/components/bookmark-button/index.tsx

```diff
@@ -24,7 +25,9 @@ export default function BookmarkButton({ isBookmarked }: BookmarkButtonProps) {
 			onClick={() => {
 				// TODO use the create / destroy methods in the client
 				// or render dialog to prompt auth if not auth'd
-				console.log('Bookmark clicked!')
+				// also, link the toast notification with the successful API post/ delete response
+				const toastAction = isBookmarked ? 'remove' : 'add'
+				makeBookmarkToast(toastAction) // TODO, pass tutorialName as second arg
```

Comment:
Will adjust this to render the tutorial name while connecting things with real data. Once we do that, we will have the tutorial name / id available in this function in order to update the API and can insert in this toast.

Reply:
I'm going to remove this invocation to avoid conflicts with #837, where there's some refactoring with the onClick handler. This PR will just serve as a stage for the `makeBookmarkToast` function for now.

Reply:
I'm going to remove this invocation and just treat this PR as a stage for the `makeBookmarkToast` function. This is to avoid future conflicts as we adjust and refactor the data layer. We will fully add this in when we integrate with the API methods.